pegasus_wrapper/artifact.py

```diff
@@ -106,16 +106,3 @@ def preexisting(
         value: _T, *, locator: Optional[Locator] = None
     ) -> "ValueArtifact[_T]":
         return ValueArtifact(value, depends_on=immutableset(), locator=locator)
-
-    @staticmethod
-    def computed(
```

Comment:
This was only used in the one test edited below.  It is not really necessary now given that the constructor can be used in the same way.